lib/filecheck/src/checker.rs

```diff
@@ -30,8 +30,8 @@ const DIRECTIVE_RX: &str = r"\b(check|sameln|nextln|unordered|not|regex):\s+(.*)
 impl Directive {
     /// Create a new directive from a `DIRECTIVE_RX` match.
     fn new(caps: Captures) -> Result<Directive> {
-        let cmd = caps.at(1).expect("group 1 must match");
-        let rest = caps.at(2).expect("group 2 must match");
+        let cmd = caps.get(1).map_or("", |m| m.as_str());
```

Comment:
I wonder if a map and and expect would more closely match(not pun intended) the original.